internal/handler/handlers.go

```diff
@@ -1986,6 +1986,21 @@ func RequestTokenSignOn(svr server.Server, userRepo *user.Repository, jobRepo *j
 			svr.JSON(w, http.StatusBadRequest, nil)
 			return
 		}
+
+		numberOfAttempts := 0
+		cachedAttempts, found := svr.CacheGet(req.Email)
```

Comment:
we need to add a prefix here so it will not conflict with future cache entries, something like `svr.CacheGet(fmt.Sprintf("sign-on-request-%s", req.Email))` same in CacheSet

Reply:
I was going for 

```go
cacheName := fmt.Sprintf("sign-on-request-%s", req.Email)
```

to be reused by both